exec/vector/src/main/codegen/templates/NullReader.java

```diff
@@ -50,6 +50,10 @@ private NullReader(MajorType type) {
   public MajorType getType() {
     return type;
   }
+
+  public void read(ValueHolder holder) {
+    throw new UnsupportedOperationException("NullReader cannot write into non-nullable holder");
```

Comment:
I'm a little bit confused here, because nullable holders also implement the ```ValueHolder``` interface.